jupyter-extension/jupyterlab_pachyderm/mount_server_client.py

```diff
@@ -12,6 +14,7 @@
 
 lock = locks.Lock()
 MOUNT_SERVER_PORT = 9002
+PACH_CONFIG_DIR = '/home/jovyan/.pachyderm/config.json'
```

Comment:
I believe the `$NB_USER` should have the current username https://jupyter-docker-stacks.readthedocs.io/en/latest/using/common.html#user-related-configurations